app/helpers/application_helper.rb

```diff
@@ -5,4 +5,10 @@ def full_title page_title = ""
     base_title = t "title"
     page_title.blank? ? base_title : "#{page_title} | #{base_title}"
   end
+
+  def convert_money_unit money, unit
```

Comment:
Đưa hàm này vào controller mà tính chứ e, sao gọi ngoài view thế

Reply:
dạ cái hàm này em gọi trong view mà a

Reply:
Ừ anh thấy em gọi ở trong view nhưng gọi trong đấy lúc render ra html nó bị chậm ấy, em gọi nó ở trong controller để tính toán gán vào biến , rồi gọi biến đó ngoài view thôi